src/utils/defaults/defaults.go

```diff
@@ -1,5 +1,6 @@
 package defaults
 
-const ProjectDirectory = "."
-const RepoName = "ghcr.io/nearform"
-const DockerfileName = "Dockerfile.kka"
+const ProjectDirectory  = "."
+const RepoName          = "ghcr.io/nearform"
+const DockerfileName    = "Dockerfile.kka"
+const DockerImage       = "gcr.io/knative-samples/helloworld-go"
```

Comment:
We don't need this, we want to deploy the image we pushed with the push command.